### PR TITLE
add AA compliance for personal info page

### PR DESCRIPTION
**Pull request by ATBull81**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [ ] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/186426648

# A brief description of the changes

Current behavior: Issues with keyboard navigation, including selecting tool tips, radio buttons, and handling ::befores

New behavior: These issues are fixed, along with contrast issues.

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name: CR97_IS_ENABLED

- [ ] DC
- [x] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

Testing will be added after this workflow is finished
Screenshot 2023-11-30 at 1 05 51 PM
Screenshot 2023-11-30 at 1 06 00 PM
Screenshot 2023-11-30 at 12 46 39 PM
**This is showing that the tooltips, that have ::before styles on them, will now still be displayed if all styling failed for the page.**

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: app/assets/stylesheets/left_nav.scss.erb [hunk 1]
```diff
@@ -211,7 +211,7 @@
   font-weight: 600;
 }
 .activer > a {
-  background: <%= EnrollRegistry[:qle_carousel].settings(:color).item %>;
+  background: var(--theme-primary-100, <%= EnrollRegistry[:qle_carousel].settings(:color).item %>);
```
Comment (mdkaraman) [inline app/assets/stylesheets/left_nav.scss.erb:214]: Is there a way to move away from using EnrollRegistry[:qle_carousel]?  Does that same color get defined in a stylesheet somewhere?  If possible, I think we should move away from referencing the CSS stored in EnrollRegistry.  Just a thought, maybe we can refactor this into `me.scss` and `dc.scss`?

File: app/views/shared/_individual_progress.html.haml [hunk 2]
```diff
@@ -40,7 +40,7 @@
 - if step == 1 && search_page == true
   = link_to "Continue", insured_consumer_role_index_path(person: @person_params), method: :post, class: 'btn btn-lg btn-primary  btn-block'
 - elsif step == 1
-  %span{class: 'btn btn-lg btn-primary btn-br btn-block', onclick: 'PersonValidations.manageRequiredValidations($(this));'} CONTINUE
+  %span{tabindex: '0', id: 'continue_button', class: 'btn btn-lg btn-primary btn-br btn-block', onkeydown: 'handleButtonKeyDown(event, "continue_button");', onclick: 'PersonValidations.manageRequiredValidations($(this));'} CONTINUE
```
Comment (mdkaraman) [inline app/views/shared/_individual_progress.html.haml:43]: Can we refactor this to `html.erb` like we did in the previous PR?

Comment (ATBull81) [reply app/views/shared/_individual_progress.html.haml:43]: I had originally refactored this, it just introduced a lot of cucumber test problems so I reverted. I can take a look again and see what parts would need to be changed.

File: app/assets/stylesheets/left_nav.scss.erb (hunk 1 above)
Comment (ATBull81) [reply app/assets/stylesheets/left_nav.scss.erb:214]: Yep, same color as a lot of the other spots (#007bc4). We can replace it everywhere we see it with the same `var(--theme-primary-100, #007bc4)` we've introduced (I still plan to rework the cr97 variable names, just planning to do that at the end of the registration workflow)

Comment (ATBull81) [reply app/assets/stylesheets/left_nav.scss.erb]: https://github.com/ideacrew/enroll/pull/3351/commits/3dfb45ff458c9ce0627f0a2df1dfce05a983fd94

File: app/views/shared/_individual_progress.html.haml (hunk 2 above)
Comment (ATBull81) [reply app/views/shared/_individual_progress.html.haml]: https://github.com/ideacrew/enroll/pull/3351/commits/046c87965fff2eec0a4c12c759819a55b67d9a0e

**Comment by ATBull81** [comment]
Wanted to have reusable onkeydown functions for buttons and radios to simulate clicks. Added here: https://github.com/ideacrew/enroll/pull/3351/commits/4e48787ee5ceb786d253afa219b73389340dbba0

**Review by mdkaraman — APPROVED** [review]
Looks good!

**Review by yosefserkez — APPROVED** [review]
looks good and clean

**Pull request merged**